pkg/kubelet/dockershim/security_context.go

```diff
@@ -143,18 +145,17 @@ func modifyContainerNamespaceOptions(nsOpts *runtimeapi.NamespaceOption, podSand
 
 // modifyCommonNamespaceOptions apply common namespace options for sandbox and container
 func modifyCommonNamespaceOptions(nsOpts *runtimeapi.NamespaceOption, hostConfig *dockercontainer.HostConfig) {
-	if nsOpts != nil {
-		if nsOpts.HostPid {
-			hostConfig.PidMode = namespaceModeHost
-		}
-		if nsOpts.HostIpc {
-			hostConfig.IpcMode = namespaceModeHost
-		}
+	if nsOpts != nil && nsOpts.HostPid {
+		hostConfig.PidMode = namespaceModeHost
 	}
 }
 
 // modifyHostNetworkOptionForSandbox applies NetworkMode/UTSMode to sandbox's dockercontainer.HostConfig.
-func modifyHostNetworkOptionForSandbox(hostNetwork bool, network *knetwork.PluginManager, hc *dockercontainer.HostConfig) {
+func modifyHostNetworkOptionForSandbox(hostNetwork bool, hostIpc bool, network *knetwork.PluginManager, hc *dockercontainer.HostConfig) {
```

Comment:
Rename this function?

Reply:
Do you have any suggestions? Perhaps just `modifyHostOptionsForSandbox` (note "Options" plural.)

A few lines below, there's `modifyHostNetworkOptionForContainer` which also sets IpcMode and UTSMode, so I'm thinking for consistency we should rename that one too... Agreed?

Reply:
I prefer `modifyHostOptionsForSandbox`. :) Since you've passed in `hostIpc` here.

Reply:
Done! Also renamed modifyHostNetworkOptionForContainer -> modifyHostOptionsForContainer for consistency (that one also sets hostIpc, even though it doesn't take it as an argument...) PTAL.